packages/components/src/components/Flex/Flex.tsx

```diff
@@ -36,16 +36,19 @@ export const flexAlignItems = [
 export type FlexDirection = (typeof flexDirection)[number];
 export type FlexJustifyContent = (typeof flexJustifyContent)[number];
 export type FlexAlignItems = (typeof flexAlignItems)[number];
+export type Flex = string;
```

Comment:
Not sure if this is anyway beneficial, but why not. One day we may figure out how to make it stricter.